go/arrow/array/fixed_size_list.go

```diff
@@ -162,25 +162,38 @@ func (a *FixedSizeList) MarshalJSON() ([]byte, error) {
 }
 
 type FixedSizeListBuilder struct {
-	builder
-
-	etype  arrow.DataType // data type of the list's elements.
-	n      int32          // number of elements in the fixed-size list.
-	values Builder        // value builder for the list's elements.
+	baseListBuilder
+	n int32 // number of elements in the fixed-size list.
```

Comment:
Stashing the length here because I couldn't figure out a way to get it from the baseListBuilder. If there is a way we can use it and drop this.